### PR TITLE
Adding a bare bones unstyled terms of service and privacy policy

### LINKED ISSUE #46: Setup a Privacy Policy
This privacy policy should follow LAION's current [Privacy Policy](https://laion.ai/privacy-policy/) but be changed to make clear what personal information gets stored and under what conditions.

When complete this should be presented in a website any user can visit.

### PR DESCRIPTION
**Pull request by fozziethebeat**
This uses the content written in Notion and provides bare bones pages for the Privacy Policy and Terms of Service using the URLs in the footer.  At a later date we can style them using whatever design system we settle on.

Closes #46

### CODE REVIEW

**Comment by fozziethebeat** [comment]
Also since this is regarding LAION as a legal entity, no merging this until Andreas (or someone Andreas delegates to) approves.

File: website/src/pages/privacy-policy.tsx
Comment (AbdBarho) [inline website/src/pages/privacy-policy.tsx:406]: makes since to extract the layout with transparent header to a component? it is used 2 times here and one for the homepage.

**Review by andreaskoepf — APPROVED** [review]
Thanks for adding it! IMO just to add it is the best way to get feedback from the relevant people, e.g. I can then simply copy the url from our dev-deployment. I assume some minor changes will finally be neccessay to the text .. but let´s start with this!

File: website/src/pages/privacy-policy.tsx
Comment (fozziethebeat) [reply website/src/pages/privacy-policy.tsx]: Done

**Comment by fozziethebeat** [comment]
Now that Andreas has approved, I'll merge once there's website approval.

**Review by AbdBarho — APPROVED** [review]

**Pull request merged**
